go/vt/vtorc/inst/analysis_dao.go

```diff
@@ -21,30 +21,26 @@ import (
 	"math"
 	"time"
 
-	"vitess.io/vitess/go/vt/external/golib/sqlutils"
-	"vitess.io/vitess/go/vt/log"
-	"vitess.io/vitess/go/vt/topo/topoproto"
-
+	"github.com/patrickmn/go-cache"
 	"google.golang.org/protobuf/encoding/prototext"
 
+	"vitess.io/vitess/go/stats"
+	"vitess.io/vitess/go/vt/external/golib/sqlutils"
+	"vitess.io/vitess/go/vt/log"
 	topodatapb "vitess.io/vitess/go/vt/proto/topodata"
 	"vitess.io/vitess/go/vt/topo"
+	"vitess.io/vitess/go/vt/topo/topoproto"
 	"vitess.io/vitess/go/vt/vtctl/reparentutil"
 	"vitess.io/vitess/go/vt/vtorc/config"
 	"vitess.io/vitess/go/vt/vtorc/db"
 	"vitess.io/vitess/go/vt/vtorc/util"
-
-	"github.com/patrickmn/go-cache"
-	"github.com/rcrowley/go-metrics"
 )
 
-var analysisChangeWriteCounter = metrics.NewCounter()
+var analysisChangeWriteCounter = stats.NewCounter("analysis.change.write", "Number of times analysis has changed")
```

Comment:
Should also not deprecate this old style naming of stats and make them consistent with Vitess-style naming?
@GuptaManan100 WDYT?

Reply:
@deepthi do you mean the `.` delimiter? If so I agree. I think most Vitess stats are `_` separated

Reply:
@deepthi I was explicitly here opting for not changing anything yet here, to not trigger a rename of metrics. That's not backwards compatible then so we'd need to probably emit both for a while and then remove the old one? 

I was here just focussed on a little cleanup. I think if we want to change the name, that's something for a separate PR?

Reply:
I think it's a good idea to make it consistent.

Reply:
> we'd need to probably emit both for a while and then remove the old one

That's what deprecation means for stats. We start emitting new stats along with old ones for 1 release, delete the old ones in a follow up release, and document in release notes of both releases.